backend/question-service/__mocks__/question.mock.ts

```diff
@@ -9,159 +9,199 @@ export const QUESTION_1: CreateQuestionDto = new CreateQuestionDto(
-    'https://leetcode.com/problems/linked-list-cycle/'
+    'https://leetcode.com/problems/linked-list-cycle/',
+    ['1->2->3->4->2', '1->2->3->4->5'],
+    ['true', 'false']
 )
 
 export const QUESTION_3: CreateQuestionDto = new CreateQuestionDto(
     'Roman to Integer',
     'Given a roman numeral, convert it to an integer.',
     [Category.ALGORITHMS],
     Complexity.EASY,
-    'https://leetcode.com/problems/roman-to-integer/'
+    'https://leetcode.com/problems/roman-to-integer/',
+    ['III', 'IV', 'IX', 'LVIII', 'MCMXCIV'],
+    ['3', '4', '9', '58', '1994']
 )
 
 export const QUESTION_4: CreateQuestionDto = new CreateQuestionDto(
     'Add Binary',
     'Given two binary strings a and b, return their sum as a binary string.',
     [Category.BIT_MANIPULATION, Category.ALGORITHMS],
     Complexity.EASY,
-    'https://leetcode.com/problems/add-binary/'
+    'https://leetcode.com/problems/add-binary/',
+    ['11', '1010', '1010', '1'],
+    ['1', '10101', '10101', '100']
 )
 
 export const QUESTION_5: CreateQuestionDto = new CreateQuestionDto(
     'Fibonacci Number',
     `The Fibonacci numbers, commonly denoted F(n), form a sequence, called the Fibonacci sequence, such that each number is the sum of the two preceding ones, starting from 0 and 1. That is,\nF(0) = 0, F(1) = 1\nF(n) = F(n - 1) + F(n - 2), for n > 1.\nGiven n, calculate F(n).`,
     [Category.RECURSION, Category.ALGORITHMS],
     Complexity.EASY,
-    'https://leetcode.com/problems/fibonacci-number/'
+    'https://leetcode.com/problems/fibonacci-number/',
+    ['2', '3', '4', '5', '6'],
+    ['1', '2', '3', '5', '8']
 )
 
 export const QUESTION_6: CreateQuestionDto = new CreateQuestionDto(
     'Implement Stack using Queues',
     'Implement a last-in-first-out (LIFO) stack using only two queues. The implemented stack should support all the functions of a normal stack (push, top, pop, and empty).',
     [Category.DATA_STRUCTURES],
     Complexity.EASY,
-    'https://leetcode.com/problems/implement-stack-using-queues/'
+    'https://leetcode.com/problems/implement-stack-using-queues/',
+    ['push', 'push', 'top', 'pop', 'empty'],
+    ['1', '2', '2', '2', 'false']
 )
 
 export const QUESTION_7: CreateQuestionDto = new CreateQuestionDto(
     'Combine Two Tables',
     `Given table Person with the following columns:\n1. personId (int)\n2. lastName (varchar)\n3. firstName (varchar)\npersonId is the primary key.\nAnd table Address with the following columns:\n1. addressId (int)\n2. personId (int)\n3. city (varchar)\n4. state (varchar)\naddressId is the primary key.\n\nWrite a solution to report the first name, last name, city, and state of each person in the Person table. If the address of a personId is not present in the Address table, report null instead. Return the result table in any order.`,
     [Category.DATABASES],
     Complexity.EASY,
-    'https://leetcode.com/problems/combine-two-tables/'
+    'https://leetcode.com/problems/combine-two-tables/',
+    [],
```

Comment:
is this supposed to be empty LOL?

Reply:
okay, now added validation to prevent empty array